@nightt5879 please fix:
```
../nuttx/tools/checkpatch.sh -c -u -m -g afc652243efd77f6050ef1c8c3a0c1a2755f4195..HEAD
❌ Missing Signed-off-by
Error: /home/runner/work/nuttx-apps/nuttx-apps/apps/examples/gps/gps_main.c:111:81: error: Long line found
Error: /home/runner/work/nuttx-apps/nuttx-apps/apps/examples/gps/gps_main.c:113:81: error: Long line found
Error: /home/runner/work/nuttx-apps/nuttx-apps/apps/examples/gps/gps_main.c:115:79: error: Long line found
Used config files:
    1: .codespellrc
Some checks failed. For contributing guidelines, see:
  https://github.com/apache/nuttx/blob/master/CONTRIBUTING.md
Error: Process completed with exit code 1.
```

--- a/examples/gps/gps_main.c
+++ b/examples/gps/gps_main.c
@@ -99,20 +99,23 @@ int main(int argc, FAR char *argv[])
 
               if (minmea_parse_rmc(&frame, line))
                 {
-                  printf("Fixed-point Latitude (NMEA * 1000)..: %"
-                         PRIdLEAST32 "\n",
+                  printf("Fixed-point Latitude................: %"
+                         PRIdLEAST32 " (NMEA * 1000)\n",
                          minmea_rescale(&frame.latitude, 1000));
-                  printf("Fixed-point Longitude (NMEA * 1000).: %"
-                         PRIdLEAST32 "\n",
+                  printf("Fixed-point Longitude...............: %"
+                         PRIdLEAST32 " (NMEA * 1000)\n",
                          minmea_rescale(&frame.longitude, 1000));
-                  printf("Fixed-point Speed (knots * 1000)....: %"
-                         PRIdLEAST32 "\n",
+                  printf("Fixed-point Speed...................: %"
+                         PRIdLEAST32 " (knots * 1000)\n",
                          minmea_rescale(&frame.speed, 1000));
-                  printf("Floating point Latitude (degrees)...: %2.6f\n",
+                  printf("Floating point Latitude.............: %2.6f "
+                         "degrees\n",
                          minmea_tocoord(&frame.latitude));
-                  printf("Floating point Longitude (degrees)..: %2.6f\n",
+                  printf("Floating point Longitude............: %2.6f "
+                         "degrees\n",
                          minmea_tocoord(&frame.longitude));
-                  printf("Floating point Speed (knots)........: %2.6f\n",
+                  printf("Floating point Speed................: %2.6f "
+                         "knots\n",
                          minmea_tofloat(&frame.speed));
                 }
               else
@@ -130,9 +133,9 @@ int main(int argc, FAR char *argv[])
                 {
                   printf("Fix quality....................: %d\n",
                          frame.fix_quality);
-                  printf("Altitude (%c)...................: %2.6f\n",
-                         frame.altitude_units ? frame.altitude_units : '?',
-                         minmea_tofloat(&frame.altitude));
+                  printf("Altitude.......................: %2.6f %c\n",
+                         minmea_tofloat(&frame.altitude),
+                         frame.altitude_units ? frame.altitude_units : '?');
                   printf("Tracked satellites.............: %d\n",
                          frame.satellites_tracked);
                 }

